modules/references-infrastructure-vspshere-failure-domains.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+// * post_installation_configuration/vsphere-failure-domain-configuration.adoc
+
+:_content-type: REFERENCE
+[id="references-infrastructure-vsphere-failure-domains_{context}"]
+= Parameters for the `infrastructures.config.openshift.io` CRD
+
+You must set values for specific parameters in the `infrastructures.config.openshift.io` Custom Resource Definition (CRD) to define multiple failure domains for your {product-title} cluster that runs on a VMware VSphere instance. For an example of a `infrastructures.config.openshift.io` configuration, see xref:../post_installation_configuration/vsphere-failure-domain-configuration.adoc#specifying-infrastructure-vsphere-failure-domains_post-install-vsphere-failure-domain-configuration[Specifying multiple failure domains for your cluster on VSphere].
```

Comment:
To be precise, change VSphere to vSphere

Reply:
Updated.